app/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/Registration.scala

```diff
@@ -73,7 +73,7 @@ case class Registration(
 
   def liabilityDetailsStatus: TaskStatus =
     if (incorpJourneyId.isEmpty)
-      TaskStatus.CannotStartYet
+      TaskStatus.NotStarted
```

Comment:
Is this the right check? Surely we should be looking to see whether we have recorded any liability details in the registration?